doc/src/explore/devnet.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Experiment with Sui DevNet
+---
+
+Welcome to the beginnings of the Sui DevNet. It exists now to gain operational experience with the Sui software in a public setting. The Sui DevNet currently consists of:
+
+* A four-validator network with all nodes operated by Mysten Labs. Clients send transactions and read requests via this endpoint: http://gateway.devnet.sui.io/
+* A public network [Sui Explorer](https://github.com/MystenLabs/sui/tree/main/explorer/client#readme) for browsing the TestNet transaction history: http://explorer.devnet.sui.io
+* A [Discord channel](https://discordapp.com/channels/916379725201563759/971488439931392130) for requesting test coins that can be used to pay for gas on the test network. These coins have no financial value and will disappear each time we reset the network.
+
+Many improvements to the Sui DevNet are underway, such as the ability to run full nodes and use a browser-based wallet. See the [Sui DevNet blog post](https://medium.com/mysten-labs) announcement for full details on upcoming features.
+
+## Purpose
+
+DevNet deployment features the following components of the Sui blockchain:
+
```

Comment:
Feels a bit weird that this section is called `Purpose`. I think it would be better if we just delete this section, and add them to the previous section

Reply:
Agreed. Replaced with Components. Just trying to split up these lists. Think modular.